src/github.com/getlantern/flashlight/android/AndroidManifest.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
We are including this file following instructions from https://github.com/golang/mobile so the user can bootstrap a test application easily.